airbyte-container-orchestrator/src/main/resources/application.yml

```diff
@@ -3,4 +3,18 @@ micronaut:
     port: 9000
 
 airbyte:
-  config-dir: /config
\ No newline at end of file
+  config-dir: /config
+  control:
+    plane:
+      auth-endpoint: ${CONTROL_PLANE_AUTH_ENDPOINT:}
+  data:
+    plane:
+      service-account:
+        credentials-path: ${DATA_PLANE_SERVICE_ACCOUNT_CREDENTIALS_PATH:}
+        email: ${DATA_PLANE_SERVICE_ACCOUNT_EMAIL:}
+  internal:
+    api:
+      auth-header:
+        name: ${AIRBYTE_API_AUTH_HEADER_NAME:}
+        value: ${AIRBYTE_API_AUTH_HEADER_VALUE:}
+      host: ${INTERNAL_API_HOST}
```

Comment:
Could you explain the purpose of this change?

Reply:
This is to be able to initialize and use AirbyteApiClient from the container orchestrator. It's a micronaut configuration file that is used when configuring the API client. An equivalent configuration is present in airbyte-workers.